src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/ExceptionInfoLookupTableNode.cs

```diff
@@ -136,9 +136,9 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
                 exceptionInfoLookupBuilder.EmitReloc(_ehInfoNode, RelocType.IMAGE_REL_BASED_ADDR32NB, _ehInfoOffsets[index]);
             }
 
-            // Sentinel record - method RVA = -1, EH info offset = total EH info size
+            // Sentinel record - method RVA = -1, EH info offset = end of the EH info block
             exceptionInfoLookupBuilder.EmitUInt(~0u);
-            exceptionInfoLookupBuilder.EmitUInt((uint)_ehInfoNode.Count);
+            exceptionInfoLookupBuilder.EmitReloc(_ehInfoNode, RelocType.IMAGE_REL_BASED_ADDR32NB, _ehInfoNode.Count);
```

Comment:
What is this reloc for? I do not see crossgen emitting anything like this.

Reply:
This is supposed to correspond to the following line:

https://github.com/dotnet/coreclr/blob/3743d9a7d5959383ae4c31b4d9273fb61e2fb389/src/zap/zapcode.cpp#L1548

My understanding of the code is that, for each CORCOMPILE_EXCEPTION_LOOKUP_TABLE_ENTRY except for the last sentinel entry, we can diff the "next ExceptionInfoRVA" and the "current ExceptionInfoRVA" to get the length of the EH clause interval corresponding to a given MethodStartRVA.